plugins/auth-backend/package.json

```diff
@@ -32,6 +32,7 @@
     "clean": "backstage-cli package clean"
   },
   "dependencies": {
+    "-": "^0.0.1",
```

Comment:
this looks a little weird?

Reply:
Agreed, remnants of some work prior to the refactoring i would imagine.